language/move-prover/bytecode/src/pipeline_factory.rs

```diff
@@ -26,6 +27,7 @@ use crate::{
 pub fn default_pipeline_with_options(options: &ProverOptions) -> FunctionTargetPipeline {
     // NOTE: the order of these processors is import!
     let mut processors: Vec<Box<dyn FunctionTargetProcessor>> = vec![
+        MutationTester::new(), // pass which may do nothing
```

Comment:
The `MutationTester` pass should not be in the default pipeline, it should be guarded with a flag `--mutation` or sth like that. I understand that if `--mutas` is 0 then the tester does nothing, but to be on the safe side, it is better to turn off the whole `MutationTester` with one flag (and by default).

Reply:
Makes sense.  Stupid question: all cases of flags enabling / disabling select another pass with and `else` block: how can I avoid that when constructing the vec?

Reply:
See how mono pass is added
```rust
    if options.run_mono {
        processors.push(MonoAnalysisProcessor::new());
    }
```